plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -408,10 +408,10 @@ function MosaicMenuItem:init()
     -- for compatibility with keyboard navigation
     -- (which does not seem to work well when multiple pages,
     -- even with classic menu)
-    self.underline_h = 1 -- smaller than default (3), don't waste space
+    self.underline_h = Size.line.thin -- smaller than default (3), don't waste space
     self._underline_container = UnderlineContainer:new{
         vertical_align = "top",
-        padding = 0,
+        padding = Size.padding.tiny,
```

Comment:
>  But if I exagerate this value and use 10, I get some drawing issues:

This was to mean that if you don't exagerate it and use 1 or 2 (that Size.padding.tiny would do), you may not notice the issue (because the shift/sliding/sizing change would be small), but it would still be there.

Reply:
> Do you really need to change this padding?

Yes, it is there to separate book covers with max possible length and width, otherwise it looks indiscernible or as if one was trying to swallow the other. Incidentally that is exactly what is happening in the original screenshots in #12132. These screenshots https://github.com/koreader/koreader/issues/12132#issuecomment-2237811434 happen to be using padding (1) and the focus happens to be in a potentially problematic book (if you were wanting the answer to the original challenge, focus on pic B was on that same Bulgakov book)